exec/java-exec/src/test/java/org/apache/drill/exec/physical/impl/writer/TestParquetWriter.java

```diff
@@ -137,14 +140,14 @@ public static void setupTestFiles() {
   public int repeat = 1;
 
   @BeforeClass
```

Comment:
For sure we can still use JUnit4 here, but interested have you tries JUnit5? 
I suppose just updating annotations is required for that, for instance `@BeforeClass` -> `@BeforeAll`..

Reply:
@vdiravka I did try to upgrade this class to JUnit5 but `@RunWith(Parameterized.class)` does not seem to have a simple direct translation (there are different translations and I do no have experience with any) so I rolled it back.